<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFidoCAD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>20000</width>
     <height>20000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QMdiArea" name="mdiArea">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="icon">
            <iconset>
             <normaloff>../../Downloads/class_libraries.ico</normaloff>../../Downloads/class_libraries.ico</iconset>
           </attribute>
           <attribute name="title">
            <string>  Libraries</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="icon">
            <iconset>
             <normaloff>../../Downloads/layers.ico</normaloff>../../Downloads/layers.ico</iconset>
           </attribute>
           <attribute name="title">
            <string> Layers</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.494, y1:0.505455, x2:0.494, y2:1, stop:0 rgba(80, 80, 80, 200), stop:1 rgba(168, 168, 168, 200));
border:1px solid black;
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="actionC_lose_All"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
    <addaction name="separator"/>
    <addaction name="actionDuplicate"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_Left"/>
    <addaction name="actionRotate_Rigth"/>
    <addaction name="separator"/>
    <addaction name="actionMirror_Horizontally"/>
    <addaction name="actionMirror_Vertically"/>
    <addaction name="separator"/>
    <addaction name="actionScale_Selected"/>
    <addaction name="separator"/>
    <addaction name="actionProperty"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionGrid_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp_online"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuView">
     <property name="title">
      <string>View</string>
     </property>
     <addaction name="actionToolBar"/>
     <addaction name="actionLibraries_Image"/>
    </widget>
    <addaction name="actionSplit_H"/>
    <addaction name="actionSplit_V"/>
    <addaction name="separator"/>
    <addaction name="actionMinimize_All"/>
    <addaction name="separator"/>
    <addaction name="menuView"/>
    <addaction name="separator"/>
    <addaction name="actionWindows_List"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.494, y1:0.505455, x2:0.494, y2:1, stop:0 rgba(80, 80, 80, 200), stop:1 rgba(168, 168, 168, 200));
border:1px solid black;</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.494, y1:0.505455, x2:0.494, y2:1, stop:0 rgba(80, 80, 80, 200), stop:1 rgba(168, 168, 168, 200));
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="actionOpen"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionGrid_Settings"/>
   <addaction name="separator"/>
   <addaction name="actionSelection"/>
   <addaction name="actionLine"/>
   <addaction name="actionText"/>
   <addaction name="actionB_zier"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionCurve"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionConnection"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/document_open.ico</normaloff>../../Downloads/document_open.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/save (4).ico</normaloff>../../Downloads/save (4).ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/document_save_as.ico</normaloff>../../Downloads/document_save_as.ico</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As ...</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionC_lose_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>C&amp;lose All</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/printer.ico</normaloff>../../Downloads/printer.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/document_new.ico</normaloff>../../Downloads/document_new.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionGrid_Settings">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/grid.ico</normaloff>../../Downloads/grid.ico</iconset>
   </property>
   <property name="text">
    <string>Grid Settings</string>
   </property>
   <property name="toolTip">
    <string>Grid Settings</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/line.png</normaloff>../../Desktop/icons32/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/rectangle.png</normaloff>../../Desktop/icons32/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/ellipse.png</normaloff>../../Desktop/icons32/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/connection.png</normaloff>../../Desktop/icons32/connection.png</iconset>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
   <property name="toolTip">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/text.png</normaloff>../../Desktop/icons32/text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
  </action>
  <action name="actionB_zier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/bezier.png</normaloff>../../Desktop/icons32/bezier.png</iconset>
   </property>
   <property name="text">
    <string>Bézier</string>
   </property>
   <property name="toolTip">
    <string>Bézier</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/polygon.png</normaloff>../../Desktop/icons32/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="toolTip">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/complexcurve.png</normaloff>../../Desktop/icons32/complexcurve.png</iconset>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
   <property name="toolTip">
    <string>Curve</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/icons32/arrow.png</normaloff>../../Desktop/icons32/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
   <property name="toolTip">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/ps_sirius_dog_black.ico</normaloff>../../Downloads/ps_sirius_dog_black.ico</iconset>
   </property>
   <property name="text">
    <string>About QFidoCAD</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/settings1.ico</normaloff>../../Downloads/settings1.ico</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/undo.ico</normaloff>../../Downloads/undo.ico</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/redo.ico</normaloff>../../Downloads/redo.ico</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/cut.ico</normaloff>../../Downloads/cut.ico</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/copy.ico</normaloff>../../Downloads/copy.ico</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/gnome_edit_paste.ico</normaloff>../../Downloads/gnome_edit_paste.ico</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/delete.ico</normaloff>../../Downloads/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionRotate_Left">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/rotate_ccw.ico</normaloff>../../Downloads/rotate_ccw.ico</iconset>
   </property>
   <property name="text">
    <string>Rotate Left</string>
   </property>
  </action>
  <action name="actionRotate_Rigth">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/rotate_cw.ico</normaloff>../../Downloads/rotate_cw.ico</iconset>
   </property>
   <property name="text">
    <string>Rotate Right</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/download (2).png</normaloff>../../Pictures/download (2).png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp_online">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/gnome_dialog_question.ico</normaloff>../../Downloads/gnome_dialog_question.ico</iconset>
   </property>
   <property name="text">
    <string>Help online</string>
   </property>
  </action>
  <action name="actionSplit_H">
   <property name="text">
    <string>Split H</string>
   </property>
  </action>
  <action name="actionSplit_V">
   <property name="text">
    <string>Split V</string>
   </property>
  </action>
  <action name="actionMinimize_All">
   <property name="text">
    <string>Minimize All</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionLibraries_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Libraries/Image</string>
   </property>
  </action>
  <action name="actionWindows_List">
   <property name="text">
    <string>Windows List</string>
   </property>
  </action>
  <action name="actionMirror_Horizontally">
   <property name="text">
    <string>Mirror Horizontally</string>
   </property>
  </action>
  <action name="actionMirror_Vertically">
   <property name="text">
    <string>Mirror Vertically</string>
   </property>
  </action>
  <action name="actionScale_Selected">
   <property name="text">
    <string>Scale Selected</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="text">
    <string>Property</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
